pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -64,6 +64,8 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 	"csi.trident.netapp.io/ontap-san": {{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock}},
 	// topolvm
 	"topolvm.cybozu.com": createTopoLVMCapabilities(),
+	// OpenStack Cinder
+	"cinder.csi.openstack.org": {{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock}},
```

Comment:
The CSI driver actually seems to sit here, so just including it for future reference
https://github.com/kubernetes/cloud-provider-openstack/tree/03db687b95ea168e0de9609a7eaad43243f88944/pkg/csi/cinder

I think we should also list FS RWO as a second preference for completeness, similarly to
https://github.com/kubevirt/containerized-data-importer/blob/208aa7ad855e40eae63c89989862b94783b2ef3c/pkg/storagecapabilities/storagecapabilities.go#L217-L218